jws.go

```diff
@@ -56,3 +58,16 @@ type staticNonceSource string
 func (s staticNonceSource) Nonce() (string, error) {
 	return string(s), nil
 }
+
+func jwkThumbprint(key rsa.PublicKey) string {
```

Comment:
I think all of the empty lines in this func can be removed. It's a simple and short function.